test/run-status.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+var path = require('path');
+var test = require('tap').test;
+var chalk = require('chalk');
+var figures = require('figures');
+var RunStatus = require('../lib/run-status');
+
+var sep = ' ' + chalk.gray.dim(figures.pointerSmall) + ' ';
+
+test('requires new', function (t) {
+	var runStatus = RunStatus;
+	t.throws(function () {
+		runStatus({});
+	}, 'Class constructor RunStatus cannot be invoked without \'new\'');
+	t.end();
+});
+
+test('prefixTitle returns empty if prefixTitles == false', function (t) {
+	var runStatus = new RunStatus({prefixTitles: false});
+	t.is(runStatus.prefixTitle('test/run-status.js'), '');
+	t.end();
+});
+
+test('prefixTitle removes base if found at start of path', function (t) {
+	var runStatus = new RunStatus({base: 'test' + path.sep});
+	t.is(runStatus.prefixTitle('test/run-status.js'), 'run-status' + sep);
+	t.end();
+});
+
+test('prefixTitle does not remove base if found but not at start of path', function (t) {
+	var runStatus = new RunStatus({base: path.sep});
```

Comment:
@novemberborn `String.prototype.replace` replaces only the first occurrence of the searched string (unless you're using a regex with `g`) so with the base `foo/` the path `foo/bar/foo/baz.js` already worked correctly before this PR — since `base` is found at the beginning and replaced the second occurrence of `base` is never considered. The only `base` for which this fails is indeed `/` because it isn't found at the beginning of the path, and thus the next occurrence of `/` was replaced